contrib/macdeploy/README.md

```diff
@@ -6,7 +6,7 @@ The `macdeployqtplus` script should not be run manually. Instead, after building
 make deploy
 ```
 
-When complete, it will have produced `Dash-Qt.dmg`.
+When complete, it will have produced `Dash-Qt.zip`.
```

Comment:
28432: it actually produces `Dash-Core.zip`

Reply:
DMG -> ZIP migration commits have been dropped per comment ([source](https://github.com/dashpay/dash/pull/6384#issuecomment-2472259081))